tests/test_operation_builder.py

```diff
@@ -420,6 +421,43 @@ def test_var_sbregion_one_block():
     assert len(op2.regs[1].blocks) == 2  # type: ignore
 
 
+@irdl_op_definition
+class TwoVarRegionOp(Operation):
+    name: str = "test.two_var_region_op"
+
+    res1: Annotated[VarRegion, StringAttr]
+    res2: Annotated[VarRegion, StringAttr]
+    irdl_options = [AttrSizedRegionSegments()]
+
+
+def test_two_var_region_builder():
```

Comment:
they can be parameterized

Reply:
with the next one

Reply:
I just tried, and I'm actually not sure how to make that work.
It seems that there is either a bug in pytest, or it is working way differently than I thought.

* I put `regionN` in a class, so I can run both "tests" with the same data. This works
* When I start using `parametrize`, this doesn't work anymore, because it is reusing the same data.

So my feeling is that `parametrize` reuse the same data across the same test, so in our case we can't make it work.